wit/timezone.wit

```diff
@@ -3,53 +3,48 @@ package wasi:clocks@0.2.0;
 @unstable(feature = clocks-timezone)
 interface timezone {
     @unstable(feature = clocks-timezone)
-    use wall-clock.{datetime};
+    use system-clock.{instant};
 
-    /// Return information needed to display the given `datetime`. This includes
-    /// the UTC offset, the time zone name, and a flag indicating whether
-    /// daylight saving time is active.
+    /// Return information needed to display the given `instant`. This includes
+    /// the UTC offset and the time zone name.
     ///
-    /// If the timezone cannot be determined for the given `datetime`, return a
-    /// `timezone-display` for `UTC` with a `utc-offset` of 0 and no daylight
-    /// saving time.
+    /// If the timezone cannot be determined for the given `instant`, return a
```

Comment:
This part should probably be removed or reworded, given that `instant`s do not have time zones.

Reply:
Now that I read it again, I'm not sure what it means for the time zone to be unable to be determined. I guess if the host doesn't have a concept of "current time zone" it should pretend the current time zone is UTC? I'll assume that's what this means, but let me know if there was a different situation in mind when writing this.